projects/schematics/package.json

```diff
@@ -22,10 +22,10 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular-devkit/core": "^10.1.0",
```

Comment:
This looks like a breaking change. Is it needed?

Reply:
Nope. I can revert it.